.github/workflows/ci.yaml

```diff
@@ -81,31 +81,26 @@ jobs:
   feature-tests-ts:
     uses: ./.github/workflows/typescript.yaml
     with:
-      typescript-repo-path: temporalio/sdk-typescript
-      typescript-repo-ref: main
+      version: 1.4.4
```

Comment:
I have opened https://github.com/temporalio/sdk-features/issues/166 for `--version latest` support. I may be able to get to it today.

I wonder, even if we add that, would it be ok to put in here? Or are there concerns of the mismatch in time between when an SDK is released and when SDK features is updated to support that release? If we're not gonna put it in here, I might not even work on the issue as it may not be worth it.

Reply:
Yeah, that's a good question. I kinda like the explicitness of this.

We could maybe do something like check if the explicit version is _not_ latest and let us know somehow? I'm just not sure what the delivery mechanism for that would be.

Reply:
Pros of explicitness:
* No downtime for SDK features because it has to be explicitly upgraded instead of being different

Cons of explicitness:
* If a release doesn't break any SDK features, people will have no reason to update this and it may get stale

Yeah, there's no good notification mechanism that can't be ignored, heh.

Maybe at the least we should pull the version out of what is set in the SDK features repo root so we don't have to remember to update there and here? Something like `--version current`? Also then you could put dependabot on this repo to tell you when to upgrade (surely you can configure it to just be for our SDKs and not all of its other noise).

Would be curious @bergundy's opinion here.

Reply:
+1 for #166, I'd just make sure that we get the concrete version when we build docker containers so we can properly tag the images.

Reply:
I'd be fine with explicit version too as long as we add a release step in our guidelines to bump here or better yet if there was a good way to automate it (assuming we have time for it).